demos/pycram_bullet_world_demo/demo.py

```diff
@@ -4,30 +4,30 @@
 from pycram.designators.object_designator import *
 from pycram.datastructures.enums import WorldMode
 from pycram.datastructures.pose import Pose
-from pycram.process_module import simulated_robot, with_simulated_robot
+from pycram.process_module import simulated_robot, with_simulated_robot, ProcessModule
 from pycram.object_descriptors.urdf import ObjectDescription
 from pycram.world_concepts.world_object import Object
 from pycram.datastructures.dataclasses import Color
 from pycram.ros_utils.viz_marker_publisher import VizMarkerPublisher
 from pycrap.ontologies import Robot, Apartment, Milk, Cereal, Spoon, Bowl
 import numpy as np
 
-
 np.random.seed(420)
 extension = ObjectDescription.get_file_extension()
 
-world = BulletWorld(WorldMode.GUI)
+world = BulletWorld(WorldMode.DIRECT)
+viz = VizMarkerPublisher()
 
 robot = Object("pr2", Robot, f"pr2{extension}", pose=Pose([1, 2, 0]))
 apartment = Object("apartment", Apartment, f"apartment{extension}")
 
-milk = Object("milk", Milk, "milk.stl", pose=Pose([2.5, 2, 1.02]),
+milk = Object("milk", Milk, "milk.stl", pose=Pose([2.5, 2, 1.02], [0, 0, 1, 1]),
```

Comment:
Isn't the whole point that the object's rotation shouldn't matter? Ideally, I'd like the base demo to maintain the initial orientation since this is our "showcase" demo at the moment.